chain-tx-filter/src/lib.rs

```diff
@@ -20,42 +19,27 @@ use std::prelude::v1::Vec;
 pub struct BlockFilter {
     // may be replaced with GCS, e.g. https://github.com/dac-gmbh/golomb-set
     bloom: Bloom,
-    modified: bool,
```

Comment:
I think modified could still be useful -- e.g. if there were no transactions processed in the enclave (there were only transactions with public information processed in abci), then it'll make sense not to attach the filter.
Perhaps -- https://github.com/crypto-com/chain/blob/master/enclave-protocol/src/lib.rs#L170 could return directly `Result<Option<(Vec<u8>, Vec<u8>)>, ()>`

Here: https://github.com/crypto-com/chain/blob/master/chain-tx-enclave/tx-validation/enclave/src/validate.rs#L31
instead of `get_raw()`, it'd call `get_tendermint_kv()`

Here: 
https://github.com/crypto-com/chain/pull/563/files#diff-bc2131504081939018fc5af67df2cc02R373

it could then use the response directly, instead of creating the filter and calling  `get_tendermint_kv()`